h/services/nipsa.py

```diff
@@ -73,23 +74,11 @@ def clear(self):
         self._flagged_userids = None
 
     def _reindex_users_annotations(self, user, tag):
-        self._get_search_index().add_users_annotations(
+        self._queue_service.queue_users_annotations(
             user.userid, tag=tag, force=True, schedule_in=30
         )
 
 
 def nipsa_factory(_context, request):
     """Return a NipsaService instance for the passed context and request."""
-
-    # NipsaService uses a search_index getter function rather than the search
```

Comment:
No need for any of this. No ES connection, we are only queuing the annotations in the postgres table.